workers/Dockerfile.tests

```diff
@@ -0,0 +1,23 @@
+FROM continuumio/miniconda3:4.3.11
+
+RUN groupadd user && useradd --create-home --home-dir /home/user -g user user
+WORKDIR /home/user
+
+COPY workers/environment.yml .
```

Comment:
This [COPY command](https://docs.docker.com/engine/reference/builder/#copy) is not just copying the file from one directory to another, but rather from the directory that the Dockerfile is being run from into the docker container. Docker containers start off completely empty so

```
RUN conda env create -f workers/environment.yml
```

wouldn't work because the container can't access a file until it's copied into the container.